src/AbstractStore.php

```diff
@@ -63,9 +63,13 @@ public function fetch($url)
 
         $ttl = 0;
         if ($res->hasHeader('Cache-Control')) {
+            $cacheControl = $res->getHeader('Cache-Control');
+            if (is_array($cacheControl)) {
```

Comment:
I don't find that function in the documentation at all. But it seems to work, and always return a string. I'll update the PR.